setup.py

```diff
@@ -21,19 +21,20 @@
         "lark-parser>=0.5.6",
         "mongogrant",
         "mongomock>=3.16",
-        "fastapi[all]",
+        "fastapi[all]>=0.31",
     ],
     extras_require={"dev": ["black", "invoke", "pre-commit", "twine"]},
     tests_require=["pytest>=3.6", "openapi-spec-validator", "jsondiff"],
     classifiers=[
         "Development Status :: 3 - Alpha",
         "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Intended Audience :: Developers",
         "Topic :: Database",
         "Topic :: Database :: Database Engines/Servers",
         "Topic :: Database :: Front-Ends",
     ],
     keywords="optimade jsonapi materials",
-    python_requires=">=3.7",
+    python_requires=">=3.6",
```

Comment:
I was under the impression we were locking in 3.7?

Reply:
I just made the change because I wanted to keep using the environment I had set up in the beginning. If there are good reasons to only allow python 3.7 that's fine with me